kpi/deployment_backends/kobocat_backend.py

```diff
@@ -664,7 +664,7 @@ def format_openrosa_datetime(dt: Optional[datetime] = None) -> str:
         return dt.isoformat('T', 'milliseconds')
 
     def duplicate_submission(
-        self, requesting_user_id: int, instance_id: int
+        self, requesting_user: 'auth.User', instance_id: int
```

Comment:
I cannot choose line 670, but there is a typo 
`Dupicates` instead of `duplicates`